src/commands/Register.ts

```diff
@@ -118,15 +118,28 @@ export class Register extends CommandClient {
 
 export async function executeRegister(registerOptions?: RegisterOptions) {
   const { app, contracts, blockchain } = getServerConfig();
-  let registerConfig: RegisterConfig | undefined = undefined;
-  if (config.has('register')) {
-    registerConfig = config.get('register');
+
+  if (!config.has('register')) {
+    throw Error(`
+    Cannot find key "register" in the configuration.
+    Make sure registration is configured in "config"
+    folder for your network, or in the default configuration.
+    `);
   }
+  const { account, stake, funds, mnemonic }: RegisterConfig =
+    config.get('register');
+
+  if (account && !mnemonic) {
+    throw Error(`
+    Cannot find key "mnemonic" in the configuration.
+    Make sure registration configuration contains "mnemonic" key 
+    for the configured "account" key in "config" folder for your network,
+    or in the default configuration.
+    `);
+  }
```

Comment:
Should we specify these parameters (`account` and `mnemonic`) in the config files? What about allowing the user to specify those parameters via command line also? I guess this should be possible by just running `npm run register -- --account="" --mnemonic=""`.

Reply:
yea, how about we use this pattern/feature https://github.com/node-config/node-config/wiki/Environment-Variables#custom-environment-variables of the config lib.

Reply:
Yes, I think it suits very well with our use case 👍🏼

Reply:
done